gax-grpc/src/test/java/com/google/api/gax/grpc/ChannelPoolTest.java

```diff
@@ -55,24 +59,25 @@
 @RunWith(JUnit4.class)
 public class ChannelPoolTest {
   @Test
-  public void testAuthority() {
+  public void testAuthority() throws IOException {
     ManagedChannel sub1 = Mockito.mock(ManagedChannel.class);
     ManagedChannel sub2 = Mockito.mock(ManagedChannel.class);
 
     Mockito.when(sub1.authority()).thenReturn("myAuth");
 
-    ChannelPool pool = new ChannelPool(Lists.newArrayList(sub1, sub2));
+    ChannelPool pool = ChannelPool.create(2, new FakeChannelFactory(Arrays.asList(sub1, sub2)));
     Truth.assertThat(pool.authority()).isEqualTo("myAuth");
   }
 
   @Test
-  public void testRoundRobin() {
+  public void testRoundRobin() throws IOException {
     ManagedChannel sub1 = Mockito.mock(ManagedChannel.class);
     ManagedChannel sub2 = Mockito.mock(ManagedChannel.class);
+
     Mockito.when(sub1.authority()).thenReturn("myAuth");
 
     ArrayList<ManagedChannel> channels = Lists.newArrayList(sub1, sub2);
-    ChannelPool pool = new ChannelPool(channels);
+    ChannelPool pool = ChannelPool.create(2, new FakeChannelFactory(Arrays.asList(sub1, sub2)));
```

Comment:
please replace 2 with channels.size() so that its obvious that the two are related. Also why not reuse the the channels list (instead of creating a new list via Arrays.asList)?